repos/prs.js

```diff
@@ -0,0 +1,48 @@
+const ReaderWriter = require('../reader-writer');
+
+module.exports = class Pulls extends ReaderWriter {
+  constructor(repos, filterFn = null) {
+    super(...arguments);
+
+    this.repos = repos;
+    this.filterFn = filterFn;
+  }
+
+  // async readCore() -- NYI
+
+  async createPRsForOneBranchName(branchName) {
+    return this.createPRs(await this.repos.map(
```

Comment:
probably good to have this also, but branches is probably going to be a tuple of branch-data & repo-data like labels ended up being.